src/CalcViewModel/Common/CalculatorDisplay.cpp

```diff
@@ -49,6 +49,18 @@ void CalculatorDisplay::SetParenDisplayText(_In_ const std::wstring& parenthesis
     }
 }
 
+void CalculatorDisplay::OnNoParenAdded()
+{
+    if (m_callbackReference != nullptr)
+    {
+        auto calcVM = m_callbackReference.Resolve<ViewModel::StandardCalculatorViewModel>();
+        if (calcVM)
+        {
+            calcVM->SetNoParenAddedNarratorAnnouncement();
```

Comment:
Consistent with the other callbacks in this file, it seems more appropriate to call `calcVM->OnNoParenAdded()` and let the ViewModel internally decide that it should update the narrator announcement.